README.md

```diff
@@ -208,105 +263,161 @@ Alternatively, to configure auth for all swagger endpoints, you can run:
+account is only used once, during the first deployment, to create the deployment service account.
+
+To manually create the utility service account:
+
+1.  In the [Google Cloud Console](https://console.cloud.google.com/), select the correct Google user account on the top
+    right and the correct GCP project in the drop down in the top center. Go to "IAM & Admin", then "Service accounts".
 
-When deploying for the first time, a Google Cloud Platform service account must first be created and credentialed.
+1.  Click "Create service account" and select "Furnish a new private key". Under "Roles", select the following
+    roles:
 
-1.  Specify the name of the Google Cloud Platform service account in `environment.local` using the variable `DSS_GCP_SERVICE_ACCOUNT_NAME`.
+    a) "Project – Owner"
 
-1.  Provision a set of credentials that will allow you to run deployment.
+    b) "Service Accounts – Service Account User"
 
-    1) In the [Google Cloud Console](https://console.cloud.google.com/), select the correct Google user account on the top
-       right and the correct GCP project in the drop down in the top center. Go to "IAM & Admin", then "Service accounts".
+    c) "Cloud Functions – Cloud Function Developer"
 
-    1) Click "Create service account" and select "Furnish a new private key". Under "Roles", select
-       a) "Project – Owner",
-       a) "Service Accounts – Service Account User"
-       a) "Cloud Functions – Cloud Function Developer".
+1.  Create the account and download the utility service account key JSON file.
 
-    1) Create the account and download the service account key JSON file.
+1.  Place the file at `$DSS_HOME/gcp-credentials-util.json`. Terraform will use this utility service account credentials
+    file to create the deployment service account.
 
-    1) Place the file as `$DSS_HOME/gcp-credentials.json`. You will replace it later.
+Now that we have the utility service account credentials, we can use Terraform to create the deployment service
+account:
+
+1.  Specify the name of the Google Cloud Platform deployment service account in `environment.local` using the environment
+    variable `DSS_GCP_SERVICE_ACCOUNT_NAME`. It should be set to `$DSS_HOME/gcp-credentials-util.json`.
+
+1.  Specify that you want to use the utility service account credentials to create the deployment service account by
+    setting `GOOGLE_APPLICATION_CREDENTIALS` to `$DSS_HOME/gcp-credentials-util.json`:
+
+    ```
+    export GOOGLE_APPLICATION_CREDENTIALS="$DSS_HOME/gcp-credentials-util.json"
+    ```
+
+1.  Create the Google Cloud Platform deployment service account using the command
 
-1.  Create the Google Cloud Platform service account using the command
     ```
     make -C infra COMPONENT=gcp_service_account apply
     ```
+
+    Alternatively, an existing service account can be imported instead using `terraform import`:
+
+    ```
```

Comment:
you have to be in `infra/google_account` dir for this to work.

Reply:
done in 411be51